functions/HttpTriggerUserIdTask/__init__.py

```diff
@@ -3,7 +3,7 @@
 import os
 import pyodbc
 import azure.functions as func
-from ..Utils.dbHandler import dbHandler
+from ..Utils.dbHandler import *
```

Comment:
was this change still needed? also with the recent changes to this file, please merge upstream dev to make sure there are no conflicts

Reply:
not exactly, I was just trying to experiment to see if thats going to destroy the code , but its shouldnt be an issue.

Reply:
Gotcha